.github/workflows/claude-code-review.yml

```diff
@@ -0,0 +1,138 @@
+name: Claude Code Review
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+permissions:
+  contents: read
+  pull-requests: write
+  issues: read
+  id-token: write
+  actions: read
+
+jobs:
+  claude-review:
+    if: github.event.pull_request.user.login == 'squadgazzz'
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Run Claude Code Review
+        uses: anthropics/claude-code-action@v1
+        env:
+          GH_TOKEN: ${{ github.token }}
+        with:
+          claude_code_oauth_token: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN }}
+          prompt: |
+            REPOSITORY: ${{ github.repository }}
+            PR NUMBER: ${{ github.event.pull_request.number }}
+            PR AUTHOR: ${{ github.event.pull_request.user.login }}
+            PR TITLE: ${{ github.event.pull_request.title }}
+
+            You are a Rust code reviewer for the CoW Protocol services codebase.
+
+            TASK:
+            1. Generate a PR description if the PR body is empty
+            2. Review the PR diff and leave inline comments on problematic code
```

Comment:
I've had success telling him to review and sumarize first, do a todo list and then start working, could that be helpful here?